api/views.py

```diff
@@ -65,7 +66,42 @@ class RegistrationAPIViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
     serializer_class = RegistrationSerializer
 
 
+class VolunteerFilter(filters.FilterSet):
+    time_volunteered = filters.NumberFilter(lookup_expr='exact')
```

Comment:
Please change all `time_volunteered` into `times_volunteered`

Reply:
It's on purpose. It doesn't work with "times" because there's already a "times_volunteered" field you return, so it's causes an error when I use it

Reply:
In that case, could you change the `times_volunteered` property into a method called `get_times_volunteered` - simply remove @property, change name and update usages (I think there's only one usage), and change `time_volunteered` into `times_volunteered`? It just feels kinda hacky, plus we're trying to mitigate English mistakes in the project which are already very common.

Reply:
No problem doing that, but it means it won't return in the results part (as it is not a property anymore)

Reply:
You're right, this is a big change that we need to do in a separate issue, we'll keep the solution you proposed in the meantime - approved.